Libraries/LibWeb/CSS/Default.css

```diff
@@ -48,7 +48,7 @@ input::placeholder, textarea::placeholder {
     line-height: initial;
 }
 
-button, input[type=submit], input[type=button], input[type=reset], select {
+button, input[type=submit], input[type=button], input[type=reset], select, ::file-selector-button {
```

Comment:
For some reason the button lost all its styles just by matching the new pseudo element, so this fixes that.

Reply:
I can't decide if that's because of a bug, or correct. :sweat_smile: Shadow styles continue to confuse me. This is good for now anyway!

Reply:
> I can't decide if that's because of a bug, or correct. 😅 Shadow styles continue to confuse me. This is good for now anyway!

[Element-backed Pseudo-Elements](https://drafts.csswg.org/css-pseudo/#element-like) says that they "often are real elements that are not otherwise selectable".
Since `::file-selector-button` doesn't say anything about it violating that rule, I'd assume that this applies here.
Then again, I think I might be missing whatever spec actually defines the shadow tree stuff inside the input element. (if that even exists)

Reply:
I would have expected this Default.css file to be loaded into Shadow DOM anyway. And the button would still match the element even if it has a pseudo attached to it.

I reckon there's a bug where an element has pseudo attached and that breaks it matching the raw element selector too or something.

Reply:
> Then again, I think I might be missing whatever spec actually defines the shadow tree stuff inside the input element. (if that even exists)

That's the fun part, there isn't a spec for it 😅 They're just vaguely defined in CSS. `::details-content` is at least defined properly now.